distribution/docs/src/main/jdocs/content/_configuring/auditing-contents.adoc

```diff
@@ -36,17 +36,18 @@ The security audit logging function does not have any configuration for audit re
 The logs themselves could be used to generate such reports outside the scope of ${branding}.
 ====
 
-//TODO DDF-3073 Implement a fallback auditing option, update and uncomment this documentation section
-////
 ===== Enabling Fallback Audit Logging
 
 * *{hardening-step}*
 
 In the event the system is unable to write to the `security.log` file, ${branding} must be configured to fall back to report the error in the application log:
 
+* edit `<INSTALL_HOME>/etc/org.ops4j.pax.logging.cfg`
+** uncomment the line (remove the `#` from the beginning of the line) for `log4j2` (`org.ops4j.pax.logging.log4j2.config.file = ${karaf.etc}/log4j2.config.xml`)
+** delete all subsequent lines
 * edit `<INSTALL_HOME>/etc/log4j2.config.xml`
-* find the entry for the `securityBackup` appender. (see example)
-* change value of `filename` and prefix of `filePattern` to the name/path of the desired failover security logs (<NEW_FILE_NAME>)
+** find the entry for the `securityBackup` appender. (see example)
+** change value of `filename` and prefix of `filePattern` to the name/path of the desired failover security logs (<NEW_FILE_NAME>)
```

Comment:
They should be able to also use a different appender as well, such as the syslog appender.